docs/operator-manual/notifications/troubleshooting-errors.md

```diff
@@ -40,6 +40,27 @@ You need to check your argocd-notifications controller version. For instance, th
 
 You have not defined `xxxx` in `argocd-notifications-cm` or to fail to parse settings.
 
+### GitHub.repoURL (\u003cno value\u003e) does not have a / using the configuration
```

Comment:
> \u003cno value\u003e

are those needed?

Reply:
This is the exact output from logs. I think it is equal to `<no value>`